mozilla-release/browser/base/content/tabbrowser.js

```diff
@@ -365,7 +365,9 @@ window._gBrowser = {
     // CLIQZ-SPECIAL:
     // DB-2208:
     // The idea to do that does not look nice but what an adventure! ;)
-    tab.setAttribute("private", window.arguments[0] === "about:privatebrowsing");
+    tab.setAttribute("private", window.arguments != null ?
```

Comment:
is this intended in this PR?

Reply:
@tsl143 No, it is not. But I put it with an additional commit intentionally.
Do you think it's better to revert it from the PR?

Reply:
Its fine